experimental/rank1_bnns/refining.py

```diff
@@ -0,0 +1,119 @@
+# coding=utf-8
+# Copyright 2020 The Edward2 Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Utilities for sampling"""
+import numpy as np
+import tensorflow.compat.v2 as tf
+import tensorflow_probability as tfp
+
+from experimental.rank1_bnns.rank1_bnn_layers import DenseRank1, Conv2DRank1
```

Comment:
Similar to the other file, can you update this to import the module, that is, `from experimental.rank1_bnns import rank1_bnn_layers`?

Reply:
Done